bin/scpipe/lib/sc/pipe.rb

```diff
@@ -19,8 +19,12 @@ def self.sclang_path
       if File.exists?("/Applications/SuperCollider.app/Contents/Resources/sclang")
         return "/Applications/SuperCollider.app/Contents/Resources/sclang"
       else
-        warn "Could not find sclang executable.\nPlease make sure that SC is either installed at the default location e.g. '/Applications/SuperCollider.app' on a mac or add sclang to your shells search path."
-        exit
+        if File.exists?("/Applications/SuperCollider.app/Contents/MacOS/sclang")
```

Comment:
Thanks for this!
Please don't open a new conditional branch here. `elsif` should be fine enough. apart from that :+1: